utils/scripts/create_parameter_json_from_nfcore_schema.py

```diff
@@ -0,0 +1,67 @@
+#!/bin/env python3
+
+"""
+Generates a parameter json for AWS HealthOmics workflows
+from NF-CORE nextflow schema JSON.
+"""
+
+from argparse import ArgumentParser
+import json
+from pprint import pprint
+import logging
+
+# Create a logger
+logging.basicConfig(format="%(name)s - %(asctime)s %(levelname)s: %(message)s")
+logger = logging.getLogger(__file__)
+logger.setLevel(logging.INFO)
+
+parser = ArgumentParser()
+parser.add_argument('--nfcore_schema_file', type=str, required=True,
```

Comment:
Where is this script intended to be run?

Nextflow schema files are typically at the root - level of a workflow definition. Suggest either:
- having a required positional argument of the workflow root, or
- having the default for this argument be `./nextflow_schema.json`

Reply:
right now the intention is quite simple, help customers generate a parameter-template.json to create workflows while migrating from nf-core. We can use your recommendation if the script is incorporated into automated solutions